src/commands/moderators/deleteAllMessage.ts

```diff
@@ -91,6 +96,14 @@ export default defineCommandHandler({
             numDeleted += userMessages.size
           } catch (error) {
             // Reply about the error
+            //for error 400 : you can bulk delete messages that are under 14 days old.
+            if ((error as DiscordAPIError).status === 400) {
+              await interaction.editReply({
+                content: `${(error as DiscordAPIError).message}`,
+                components: [],
+              })
+              continue
+            }
             await interaction.editReply({
               content: `Error deleting messages: ${
                 (error as DiscordAPIError).message
```

Comment:
Please also change any existing like this.

Reply:
Thank you and I think i will re-structure (try-catch) to more readable.